pkg/kubectl/proxy_server.go

```diff
@@ -34,7 +34,7 @@ import (
 
 const (
 	DefaultHostAcceptRE   = "^localhost$,^127\\.0\\.0\\.1$,^\\[::1\\]$"
-	DefaultPathAcceptRE   = "^/.*"
+	DefaultPathAcceptRE   = "^.*"
```

Comment:
Will this accept `localhost:8001path` without the `/` between host and path?
WDYT of this: `"^$|^/.*"`